uint32 id
uint32 source_id
uint32 destination_id
duration duration
geometry_msgs/Transform transform